selenium_tests/cross_listing/page_objects/cross_listing_base_page_object.py

```diff
@@ -0,0 +1,34 @@
+from selenium.common.exceptions import NoSuchElementException
+from selenium.webdriver.common.by import By
+
+from selenium_common.base_page_object import BasePageObject
+
+
+class Locators(object):
+    @classmethod
+    def TD_TEXT_XPATH(cls, search_text):
```

Comment:
Related PR for moving this into selenium common here:
https://github.com/Harvard-University-iCommons/selenium_common/pull/16